File: Python/classes_Objects/person.py
class Person:

   def __init__(self, name, age):
       self.name = name
       self.age = age


   def print_details(self):
     return f"{self.name } \n{self.age}"



if __name__ == "__main__":
    Alice = Person("Alice", 25)
    Bob = Person("Bob" , 30)

    # print(Alice.name)
    print(Alice.print_details())

File: Java/inheritance/Person.java
public class Person {
    private String name;
    private int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    void introduce(){
        System.out.println("Hi , Iam " + name + "and iam " + age + "Years ");
    }
}

class Student extends Person {
   private String school;

    public Student(String name, int age, String school) {
        super(name, age);
        this.school = school;
    }

    void study(){
        System.out.println("Iam studing in " + school);
    }
}
